### PR TITLE
LPS-128835 active filters are now highlighted on dataset display filters dropdown

### PR DESCRIPTION
**Pull request by FabioDiegoMastrorilli**
Just a trivial fix. Filters didn't update in the dataset display management bar; now they do.

I've also enforced some type checking: radio filters and numbers didn't work well when an item value was === 0.

### CODE REVIEW

**Comment by FabioDiegoMastrorilli** [comment]
ci:test:sf

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/FiltersDropdown.js [hunk 1]
```diff
@@ -25,17 +25,20 @@ function FiltersDropdown() {
 	const [active, setActive] = useState(false);
 	const [query, setQuery] = useState('');
 
-	const [visibleFilters] = useState(
-		filtersState.filters.filter((filter) => !filter.invisible)
+	const visibleFilters = useMemo(
+		() => filtersState.filters.filter((filter) => !filter.invisible),
```
Comment (wincent) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/FiltersDropdown.js:29]: Can haz more instances of the string "filter" on this line plz? 😜

**Review by wincent — COMMENTED** [review]
I'm a bit too tired to fully understand what's going on here (it's subtle), but I don't see anything glaringly wrong with it.

best-i-can-do

One thing to note: I think the bot might reject this PR because the PR title and commits don't start with one of the "blessed" prefixes (IFI is not blessed, as far as I know; you need to create an LPS for this).

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/FiltersDropdown.js (hunk 1 above)
Comment (FabioDiegoMastrorilli) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/FiltersDropdown.js:29]: I'm trying my best to win a secret contest :P

**Comment by wincent** [comment]
@FabioDiegoMastrorilli: I don't know if the test failures are legit but at least the SF needs a fix before we try re-running, so you may as well rebase at the same time.

**Comment by jbalsas** [comment]
Couldn't push, probably because I'm bad at git and ssh. Since I ran out of time to try to figure it out, I've created https://github.com/liferay-frontend/liferay-portal/pull/881 😭

**Comment by FabioDiegoMastrorilli** [comment]
@jbalsas and @wincent thanks a lot, I've been off for 2 days and I didn't check what was going on here

**Comment by jbalsas** [comment]
> I've been off for 2 days and I didn't check what was going on here

Exactly as you shouldn't have 👍

**Pull request closed without merging**